docs/source/recipes/aishell/stateless_transducer.rst

```diff
@@ -481,7 +481,7 @@ After downloading, you will have the following files:
   |   |-- log-decode-epoch-64-avg-33-context-2-max-sym-per-frame-1-2022-02-28-18-13-07
   |   |-- recogs-test-beam_4-epoch-64-avg-33-beam-4.txt
   |   `-- recogs-test-greedy_search-epoch-64-avg-33-context-2-max-sym-per-frame-1.txt
-  `-- test_wavs
+  `-- test_waves
```

Comment:
Sorry, this one is correct. Please see
https://huggingface.co/csukuangfj/icefall-aishell-transducer-stateless-modified-2022-03-01/tree/main/test_wavs

Reply:
Could you fix this one?